app/src/main/res/values/strings.xml

```diff
@@ -333,4 +333,5 @@
     <string name="dialogs_top_nine_month">Top Nine Months</string>
     <string name="post_listing_view_source">View Source</string>
     <string name="post_listing_view_original">View Original</string>
+    <string name="back_warning">Repeat action to exit</string>
```

Comment:
Are there are other actions besides pressing the back button/gesture equivalent that we would need to be concerned about? If not, "Press back again to exit" sounds more natural to me. Otherwise LGTM.

Reply:
Yeah I didn't want to put tap because of the back gesture, but that could work too.

Reply:
Ya I also like "Press back again to exit"